### PR TITLE
docs: recommend tooling

### LINKED ISSUE #181: JLBP-11: Add recommendations for tooling
Staying up to date with dependencies is much easier with tooling, so we should make some recommendations in https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/library-best-practices/JLBP-11.md .

### LINKED ISSUE #908: Consider a renovabot recommendation in JLBP-11
Or other tooling for keeping dependencies up to date.

### PR DESCRIPTION
**Pull request by elharo**
fix #181 
fix #908

### CODE REVIEW

File: docs/JLBP-11.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
...
+Staying up to date is also important to ensure that security fixes are rolled
+out promptly.
+
+There are several tools that make upgrading dependencies easier by performing much of the grunt work:
+
+* The [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/)
+  can inform you of new versions of dependencies and plugins or rewrite a
+  pom.xml to use the new versions.
+* [Renovabot](https://renovate.whitesourcesoftware.com/) notices when a new version of a dependency is released and files a pull request against your Github repository.
+* Github's [Dependabot](https://dependabot.com/) creates pull requests to address security advisories and dependency upgrades.
```
Comment (suztomo) [inline docs/JLBP-11.md]:
```suggestion
* GitHub's [Dependabot](https://dependabot.com/) creates pull requests to address security advisories and dependency upgrades.
```

File: docs/JLBP-11.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
...
+
+These tools all assume that the new versions are compatible with the older versions. In cases where they're not fully compatible a depdency upgrade can still require manual attention to rewrite and update affected code. However these tools do remove much of the pain from small updates.
```
Comment (suztomo) [inline docs/JLBP-11.md]: Would you set the line width at 100 characters?

File: docs/JLBP-11.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
...
 more likely to be internally consistent and compatible.
+Staying up to date is also important to ensure that security fixes are rolled
+out promptly.
+
+There are several tools that make upgrading dependencies easier by performing much of the grunt work:
+
+* The [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/)
+  can inform you of new versions of dependencies and plugins or rewrite a
+  pom.xml to use the new versions.
+* [Renovabot](https://renovate.whitesourcesoftware.com/) notices when a new version of a dependency is released and files a pull request against your Github repository.
```
Comment (suztomo) [inline docs/JLBP-11.md]:
```suggestion
* [Renovate Bot](https://renovate.whitesourcesoftware.com/) notices when a new version of a dependency is released and suggests changes for your project.
```

Or Renovatebot, or Renovate.

Renovate Bot as GitHub App is one form of Renovate. Renovate also works for GitLab and on-premise projects. Pull request is GitHub's term.

File: docs/JLBP-11.md (hunk 1 above)
Comment (elharo) [reply docs/JLBP-11.md]: done

File: docs/JLBP-11.md (hunk 3 above)
Comment (elharo) [reply docs/JLBP-11.md]: done

File: docs/JLBP-11.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
 releases a higher version. Time is important, not just how many versions behind,
 because a dependency tree that is up to date at each point in time is
 more likely to be internally consistent and compatible.
+Staying up to date is also important to ensure that security fixes are rolled
+out promptly.
+
+There are several tools that make upgrading dependencies easier by performing much of the grunt work:
+
+* The [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/)
+  can inform you of new versions of dependencies and plugins or rewrite a
```
Comment (garrettjonesgoogle) [inline docs/JLBP-11.md]: For readability: "or rewrite" -> "or it can rewrite"

File: docs/JLBP-11.md [hunk 5]
```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
...
+  pom.xml to use the new versions.
+* [Renovate](https://renovate.whitesourcesoftware.com/) notices when a new version of a dependency is released and files a pull or merge request to update your repository.
+* GitHub's [Dependabot](https://dependabot.com/) creates pull requests to address security advisories and dependency upgrades.
+
+These tools assume that the new versions are compatible with the older versions. In cases where they're not fully compatible, a depdency upgrade can still require manual attention to rewrite and update affected code. However these tools do remove much of the pain from small updates.
```
Comment (garrettjonesgoogle) [inline docs/JLBP-11.md]: Spelling (depdency)

Comment (elharo) [reply docs/JLBP-11.md]: done

File: docs/JLBP-11.md (hunk 4 above)
Comment (elharo) [reply docs/JLBP-11.md]: rewrote

File: docs/JLBP-11.md [hunk 6]
```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
...
+
+There are several tools that make upgrading dependencies easier by performing much of the grunt work:
+
+* The [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/)
+  can inform you of new versions of dependencies or rewrite a
+  pom.xml to use the new versions.
+* [Renovate](https://renovate.whitesourcesoftware.com/) notices when a new version of a dependency is released and files a pull or merge request to update your repository.
+* GitHub's [Dependabot](https://dependabot.com/) creates pull requests to address security advisories and dependency upgrades.
+
+These tools assume that the new versions are compatible with the older versions. In cases where they're not fully compatible, a dependency upgrade can still require manual attention to rewrite and update affected code. However these tools do remove much of the pain from small updates.
```
Comment (suztomo) [inline docs/JLBP-11.md]: Would you set the line width no more than 100 characters?

File: docs/JLBP-11.md [hunk 7]
```diff
@@ -4,6 +4,21 @@ Release no later than 6 weeks after any of your dependencies
...
 more likely to be internally consistent and compatible.
+Staying up to date is also important to ensure that security fixes are rolled
+out promptly.
+
+There are several tools that make upgrading dependencies easier by performing much of the grunt work:
+
+* The [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/)
+  can inform you of new versions of dependencies or rewrite a
+  pom.xml to use the new versions.
+* [Renovate](https://renovate.whitesourcesoftware.com/) notices when a new version of a dependency is released and files a pull or merge request to update your repository.
```
Comment (suztomo) [inline docs/JLBP-11.md]: This line is also long.

**Pull request merged**